### PR TITLE
Ci pylint fixes

### PR DESCRIPTION
**Pull request by tobiaskleiner**
Adding the fixes here for the updated pylint version. Copied over from the light-emission branch.

### CODE REVIEW

File: simtools/applications/tune_psf.py [hunk 1]
```diff
@@ -276,8 +278,11 @@ def run_pars(pars, plot=True):
         im = ray.images()[0]
         d80 = im.get_psf()
 
-        # Simulated cumulative PSF
-        data_to_plot["simulated"] = im.get_cumulative_data(radius * u.cm)
+        if radius is not None:
```
Comment (GernotMaier) [inline simtools/applications/tune_psf.py:281]: I think we violate here our aim of 'locality': the error should be thrown in get_cumulative_data.

File: simtools/applications/tune_psf.py [hunk 2]
```diff
@@ -313,22 +318,24 @@ def run_pars(pars, plot=True):
...
-    run_pars(best_pars, plot=True)
-
-    plt.close()
-    pdf_pages.close()
-
-    # Printing the results
-    print("Best parameters:")
-    for par, value in best_pars.items():
-        print(f"{par} = {value}")
+    if best_pars is not None:
```
Comment (GernotMaier) [inline simtools/applications/tune_psf.py:327]: Same thing there, as run_pars apparently cannot handle best_pars == None, the error handling should happen there.

**Review by GernotMaier — CHANGES_REQUESTED** [review]
Two relatively minor things. Thanks for fixing!

File: simtools/applications/tune_psf.py (hunk 1 above)
Comment (tobiaskleiner) [reply simtools/applications/tune_psf.py]: Yes right, I removed it. It's already inside `get_cumulative_data`.

File: simtools/applications/tune_psf.py (hunk 2 above)
Comment (tobiaskleiner) [reply simtools/applications/tune_psf.py]: Changed

**Review by GernotMaier — APPROVED** [review]
All good, thanks!

**Comment by GernotMaier** [comment]
To add here: this PR is necessary due to increased sensitivity of pylint for some issues with version 3.2

**Comment by tobiaskleiner** [comment]
Thanks for the review! Merging now.

**Pull request merged**